packages/forms/frontend/sirius-components-forms/src/propertysections/__tests__/ListPropertySection.test.tsx

```diff
@@ -10,98 +10,26 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { MockedProvider, MockedResponse } from '@apollo/client/testing';
+import { MockedProvider } from '@apollo/client/testing';
 import {
   MessageOptions,
   Selection,
   SelectionContext,
   ToastContext,
   ToastContextValue,
 } from '@eclipse-sirius/sirius-components-core';
-import { act, cleanup, render, screen, waitFor } from '@testing-library/react';
+import { cleanup, render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { afterEach, expect, test, vi } from 'vitest';
-import { GQLList, GQLListItem } from '../../form/FormEventFragments.types';
-import { ListPropertySection, clickListItemMutation } from '../ListPropertySection';
-import {
-  GQLClickListItemMutationData,
-  GQLClickListItemMutationVariables,
-  GQLSuccessPayload,
-} from '../ListPropertySection.types';
+import { ListPropertySection } from '../ListPropertySection';
+import { listReadOnly, list, itemClickCalledCalledSuccessMock } from './ListPropertySection.data';
 
 const setSelection = () => {};
 
-crypto.randomUUID = vi.fn(() => '48be95fc-3422-45d3-b1f9-d590e847e9e1');
-
-afterEach(() => cleanup());
-
-const defaultListItems: GQLListItem[] = [
-  {
-    id: 'item1',
-    label: 'item1Label',
-    kind: 'itemKind',
-    iconURL: [],
-    deletable: true,
-  },
-  {
-    id: 'item2',
-    label: 'item2Label',
-    kind: 'itemKind',
-    iconURL: [],
-    deletable: true,
-  },
-  {
-    id: 'item3',
-    label: 'item3Label',
-    kind: 'itemKind',
-    iconURL: [],
-    deletable: true,
-  },
-];
-const defaultList = {
-  label: 'myList',
-  iconURL: [],
-  hasHelpText: false,
-  items: defaultListItems,
-  style: null,
-  __typename: 'List',
-  diagnostics: [],
-  id: 'listId',
-  readOnly: false,
-};
-
-const defaultListWithStyle: GQLList = {
-  ...defaultList,
-  style: {
-    color: 'RebeccaPurple',
-    bold: true,
-    fontSize: 20,
-    italic: true,
-    strikeThrough: true,
-    underline: true,
-  },
-};
-
-const readOnlyList: GQLList = {
-  ...defaultListWithStyle,
-  readOnly: true,
-};
-
-const clickListItemVariables: GQLClickListItemMutationVariables = {
-  input: {
-    id: '48be95fc-3422-45d3-b1f9-d590e847e9e1',
-    editingContextId: 'editingContextId',
-    representationId: 'formId',
-    listId: 'listId',
-    listItemId: 'item3',
-    clickEventKind: 'SINGLE_CLICK',
-  },
-};
-
-const successPayload: GQLSuccessPayload = { messages: [], __typename: 'SuccessPayload' };
-const clickListItemSuccessData: GQLClickListItemMutationData = {
-  clickListItem: successPayload,
-};
+afterEach(() => {
+  cleanup();
+  vi.clearAllMocks();
```

Comment:
Shouldn't we need this everywhere we have mocks? For example, in button property section tests, etc